kasa/tests/test_cli.py

```diff
@@ -149,7 +149,15 @@ async def test_command_with_child(dev, mocker):
     runner = CliRunner()
     update_mock = mocker.patch.object(dev, "update")
 
-    dummy_child = mocker.create_autospec(IotDevice)
+    # create_autospec for device slows tests way too much, so we use a dummy here
```

Comment:
Did you try profiling to see which part is making it slow? I'm just wondering if the root cause is outside the autospec and in some part of the lib.

Reply:
I didn't originally as there was a comment elsewhere about it so I assumed it was.  I just tried to `create_autospec` on the `DummyChild` class and it was slow again so I think the issue is with `create_autospec` rather than the lib.